packages/storefront-lwc/src/modules/commerce/lineitemtotal/lineitemtotal.html

```diff
@@ -0,0 +1,21 @@
+<!--
```

Comment:
Components should follow the naming conventions.

Reply:
done